### DISCUSSION TITLE
Spaces are inserted around operators in `calc()` but not in `max()`?

### DISCUSSION THREAD

**Discussion by PJBaker** [tailwindlabs/tailwindcss#8605] [Help]
I sometimes forget to put spaces around operators in `calc()`, so I like the fact that Tailwind automatically inserts them for me. However, I've noticed that it does not do the same inside other functions like `max()` - is this intentional?

For example, these all work:

    w-[calc(100%_-_32px)]
    w-[calc(100%-32px)]
    w-[max(100%_-_32px,320px)]

but this doesn't:

    w-[max(100%-32px,320px)]

The cause of this seems to be this code, which is specific to `calc`, but perhaps should work for all `cssFunctions`:

https://github.com/tailwindlabs/tailwindcss/blob/922191450b9d36b5b2bc600bccdce7424ee5271a/src/util/dataTypes.js#L4
https://github.com/tailwindlabs/tailwindcss/blob/922191450b9d36b5b2bc600bccdce7424ee5271a/src/util/dataTypes.js#L43-L50

**Comment by adamwathan** [comment]
Yeah this is just something we don’t do because we haven’t done it, haha. Agree would be nice to add! Open to a PR if interested in giving it a shot 👍🏻
